container/features/pom.xml

```diff
@@ -85,9 +85,64 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+          <groupId>org.apache.karaf.tooling</groupId>
+          <artifactId>karaf-maven-plugin</artifactId>
+          <version>${karafVersion}</version>
+          <extensions>true</extensions>
+          <configuration>
+            <skip>${skipTests}</skip>
+          </configuration>
+          <executions>
+              <execution>
+                  <id>verify</id>
+                  <phase>process-resources</phase>
+                  <goals>
+                      <goal>verify</goal>
+                  </goals>
+                  <configuration>
+                      <javase>11</javase>
+                      <descriptors>
+                          <descriptor>mvn:org.apache.karaf.features/framework/${karafVersion}/xml/features</descriptor>
+                          <descriptor>mvn:org.apache.karaf.features/standard/${karafVersion}/xml/features</descriptor>
+                          <descriptor>file:${project.build.directory}/classes/features.xml</descriptor>
+                          <descriptor>file:${project.build.directory}/classes/features-minion.xml</descriptor>
+                          <descriptor>file:${project.build.directory}/classes/spring.xml</descriptor>
+                      </descriptors>
+                      <distribution>org.apache.karaf.features:framework</distribution>
+                      <framework>
+                          <feature>framework</feature>
+                          <feature>aries-blueprint</feature>
+                      </framework>
+                      <features>
+                          <feature>minion-api-layer</feature>
```

Comment:
This plugin makes it possible to test that features are installable and contain all the required bundles. Works reliably too!